main.cpp

```diff
@@ -51,10 +51,22 @@ void StartGame()
 
 }
 
+
 void Meniusetup(RenderWindow &meniu)
 {
 
-    Text startText,startText,exitText,settingText,soundText;
+   sf::Sprite sprite;
+   sf::Texture texture;
+    if(!texture.loadFromFile("Forma.jpg"))

```

Comment:
In folder-ul Assets, fa un folder Images, si baga Forma.jpg acolo. Incarc-o de acolo folosind pathSeparator, cum ai exemplu pe master